### PR TITLE
MACOSX: Put documentation links in Help menu

### PR DESCRIPTION
**Pull request by ccawley2011**
> The local documents are not currently internationalised simply
because the internationalised resources are not put into the right
places (NSBundle will handle this automatically when they are);
Trac#10464 is a tracking bug for this outstanding issue.

Originally from PR #1296.

### CODE REVIEW

**Comment by criezy** [comment]
Since a ScummVM build might stay around for a long time, my guess is that the idea is to use a path that should be persistent and redirect to wherever our user manual is. Currently it is on the wiki, but this might change.

**Comment by lotharsm** [comment]
@criezy, I suggest creating either a subdomain manual.scummvm.org or something like scummvm.org/help that we then redirect to whatever the current location of the User Manual is.

**Comment by Mataniko** [comment]
I'm planning to have something at docs.scummvm.org eventually. In the meanwhile, we already have scummvm.org/documentation which links to both the README and the Wiki.

File: ports.mk [hunk 1]
```diff
@@ -111,15 +111,20 @@ ifdef USE_SPARKLE
...
 ifdef DIST_FILES_ENGINEDATA
 	cp $(DIST_FILES_ENGINEDATA) $(bundle_name)/Contents/Resources/
 endif
-	$(srcdir)/devtools/credits.pl --rtf > $(bundle_name)/Contents/Resources/Credits.rtf
+	$(srcdir)/devtools/credits.pl --rtf > $(bundle_name)/Contents/Resources/AUTHORS.rtf
+	rm $(bundle_name)/Contents/Resources/AUTHORS
+	echo "\n---\n" | cat - $(bundle_name)/Contents/Resources/COPYING.LGPL >> $(bundle_name)/Contents/Resources/COPYING
+	echo "\n---\n" | cat - $(bundle_name)/Contents/Resources/COPYING.FREEFONT >> $(bundle_name)/Contents/Resources/COPYING
+	echo "\n---\n" | cat - $(bundle_name)/Contents/Resources/COPYING.BSD >> $(bundle_name)/Contents/Resources/COPYING
+	rm $(bundle_name)/Contents/Resources/COPYING.LGPL $(bundle_name)/Contents/Resources/COPYING.FREEFONT $(bundle_name)/Contents/Resources/COPYING.BSD
```
Comment (angstsmurf) [inline ports.mk]: I am not a lawyer, but perhaps a simple way to comply with the comments made by @sev- [here](https://github.com/scummvm/scummvm/pull/1296#issuecomment-414053875) would be to remove this line and keep all the COPYING files unmodified inside the bundle, and instead create a fifth file, call it something like COPYING-COMBINED, concatenate all the license files into it, and link the Licensing Information menu item to this. In this way the license files would still be distributed in unmodified form along with the binary. Something like [this](https://github.com/angstsmurf/scummvm/commit/62d29e1ed81c2b0c7b9df0a1dfef6186554a2fcc).

An alternative would be to create three new menu items, one for every license file.

**Comment by bluegr** [comment]
This looks to be OK to be merged

**Comment by sev-** [comment]
Merging, thanks!

**Pull request merged**